packages/@coorpacademy-components/src/molecule/teams-popin/style.css

```
@@ -0,0 +1,102 @@
+@value colors: "../../variables/colors.css";
+@value dark from colors;
+@value brand from colors;
+
+
+.disable {
+  top: 0px;
+  position: relative;
+  width: 100%;
+  margin: 0;
+  padding: 15% 25%;
+  background-color: '#eceff1';
```

`@value light from colors;`
```suggestion
  background-color: light;
```